vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -357,11 +359,39 @@ func newBucket(
 	url *url.URL,
 	userAgent string,
 	name string,
-	billingProject string) (b Bucket, err error) {
+	billingProject string,
+	tokenSrc oauth2.TokenSource,
+	goClientConfig *GoClientConfig) (b Bucket, err error) {
 
 	// Creating client through Go Storage Client Library for the storageClient parameter of bucket.
+	var tr *http.Transport = nil
+
+	// Choosing between HTTP1 and HTTP2.
+	if goClientConfig.DisableHTTP2 {
+		// Disables HTTP 2.0.
```

Comment:
Avi meant add this comment just above setting TLSNextProto , because setting TLSNextProto is what disables http2.

like this: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/main.go#L109

Reply:
Done